video2dataset/dataloader/video_decode.py

```diff
@@ -44,6 +44,7 @@ class VideoDecorder(AbstractVideoDecoder):
     def __init__(
         self,
         n_frames=None,
+        subsample_frames=None,
```

Comment:
can we rename this to something more clear? in video2dataset "subsample" generally means a transformation that reduces dimensionality. I think here you mean something like "evenly_sample" or something like that. Am I understanding this code correctly?